### PR TITLE
Add fast_ec_add hint test

### PR DESCRIPTION
**Pull request by mmsc2**

### CODE REVIEW

**Review by pefontana — CHANGES_REQUESTED** [review]

File: cairo_programs/fast_ec_add.cairo [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+end
+
+
+func main{range_check_ptr: felt}():
+
+    let x = BigInt3(1, 5, 10)
+    let y = BigInt3(2, 4, 20)
+
+    let point_a = EcPoint(x, y)
+    let (point_e) = ec_double(EcPoint(BigInt3(156, 6545, 100010), BigInt3(-5336262886225868, 1324, -113834066795297906)))
```
Comment (pefontana) [inline cairo_programs/fast_ec_add.cairo:78]: Lets avoid using the ec_double( function we are not testing here) to create a variable, just define point_e as` point_e = EcPoint(....)`
You can take the paremeters from `cairo_programs/secp_ec.cairo` in cairo_rs

Comment (pefontana) [reply cairo_programs/fast_ec_add.cairo:78]: @mmsc2

File: hints_tests.py [hunk 2]
```diff
@@ -52,6 +52,8 @@ def test_program(program_name: str):
     test_program("signed_div_rem")
     test_program("find_element")
     test_program("search_sorted_lower")
+    # test_program("set_add") # Custom Hint Error: AttributeError: 'builtins.PyMemory' object has no attribute 'get_range'
```
Comment (pefontana) [inline hints_tests.py:55]: Don't comment the set_add test

**Review by pefontana — APPROVED** [review]

File: cairo_programs/fast_ec_add.cairo (hunk 1 above)
Comment (pefontana) [reply cairo_programs/fast_ec_add.cairo:78]: Don't worry about this, it is not really necessary

**Review by entropidelic — APPROVED** [review]

**Pull request merged**